src/homework/calendar/CalendarNew.java

```diff
@@ -9,14 +9,14 @@
     public static void main(String[] args) {
 
         LocalDate todayDate = LocalDate.now();
-
-        //int actYear = todayDate.getYear();
-        //int actMonth = todayDate.getMonthValue();
         int actDay = todayDate.getDayOfMonth();
 
-        int daysInMonth = todayDate.lengthOfMonth();
+        //int daysInMonth = todayDate.lengthOfMonth();
+
+        System.out.println("\n" + Month.of(todayDate.getMonthValue()).toString() + " " + todayDate.getYear() + "\n");
```

Comment:
`System.out.format()` - rozdziela warstwę prezentacji od danych
`System.out.format("\n%s %s\n", Month.of(todayDate.getMonthValue()).toString(), todayDate.getYear() );`